sdk/appcenter/src/main/java/com/microsoft/appcenter/utils/NetworkStateHelper.java

```diff
@@ -62,15 +63,15 @@
     /**
      * Current network state.
      */
-    private boolean mConnected;
+    private final AtomicBoolean mConnected = new AtomicBoolean();
 
     /**
      * Init.
      *
      * @param context any Android context.
      */
     @VisibleForTesting
-    NetworkStateHelper(Context context) {
+    public NetworkStateHelper(Context context) {
```

Comment:
Since this becomes a public method, you can remove the annotation for `VisibleForTesting`.

Reply:
It's public only because of tests I think so it's actually what this annotation is used for.

Reply:
Ah.. It is used in unit tests in different packages. I see.